### PR TITLE
2307 sweep mode

### PR DESCRIPTION
**Pull request by YeChen-IDM**

### CODE REVIEW

File: run_sims/sweeps/historical_archetype_sweeps.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+from functools import partial
+
+from run_sims.archetypes.magude.magude_campaign import build_full_magude_campaign
+from run_sims.archetypes.maka.maka_campaign import build_full_maka_campaign
```
Comment (YeChen-IDM) [inline run_sims/sweeps/historical_archetype_sweeps.py:4]: maka_campaign is not checked in this PR.

File: _preliminary/sweep_over_habitat_to_get_prevalence.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
+from functools import partial
+
+from emodpy.emod_task import EMODTask
+from idmtools.builders import SimulationBuilder
+from idmtools.core.platform_factory import Platform
+from idmtools.entities.experiment import Experiment
+
+from run_sims import manifest
+from run_sims.build_config import set_full_config
+from run_sims.other import include_post_processing
```
Comment (YeChen-IDM) [inline _preliminary/sweep_over_habitat_to_get_prevalence.py:10]: run_sims.other is not checked in.

Comment (jgsuresh) [reply _preliminary/sweep_over_habitat_to_get_prevalence.py]: I renamed this file to "helpers", and have changed the code accordingly

File: run_sims/sweeps/historical_archetype_sweeps.py (hunk 1 above)
Comment (jgsuresh) [reply run_sims/sweeps/historical_archetype_sweeps.py:4]: I see maka_campaign.py in my branch.. can you double check that you don't have it in yours?

Comment (YeChen-IDM) [reply run_sims/sweeps/historical_archetype_sweeps.py:4]: I can see it now, thank you Josh!

**Comment by YeChen-IDM** [comment]
I did a "test" archetype run and simulation failed with this error:
```
GeneralConfigurationException: 
Exception in utils/BaseEventTriggerTemplates.h at 293 in CreateTrigger.
'Event_Trigger_List', a INDIVIDUAL event type, with specified value 'Received_Treatment' is invalid.
Custom INDIVIDUAL events must be defined in the Custom_Individual_Events list.
The built-in plus custom events are:
```
Here is my simulation in Comps: https://comps.idmod.org/#explore/Simulations?filters=ExperimentId=8b170d55-3d32-ee11-aa09-b88303911bc1&offset=0&count=100&selectedId=1f399569-3d32-ee11-aa09-b88303911bc1&resizer=472C262C0

File: run_sims/reports.py [hunk 3]
```diff
@@ -1,15 +1,20 @@
...
+        report_start_day = 1
 
     add_malaria_cotransmission_report(emod_task, manifest=manifest,
                                       start_day=report_start_day, max_age_years=1000)
     add_report_infection_stats_malaria(emod_task, manifest=manifest,
-                                       start_day=report_start_day, reporting_interval=reporting_interval)
+                                       start_day=report_start_day, reporting_interval=30)
+    # if include_debugging_reports:
+    add_report_event_counter(emod_task, manifest=manifest,
+                             start_day=report_start_day, event_trigger_list=["InfectionDropped","Received_Treatment"])
```
Comment (YeChen-IDM) [inline run_sims/reports.py:20]: some of the archetypes has only importation intervention, we may need to update the event_trigger_list according to the archetypes. For example, test and maka will not have "Received_Treatment" event.

Comment (jgsuresh) [reply run_sims/reports.py:20]: Yes!! Good find, thank you Ye. I made a change that should resolve this issue

**Pull request merged**